zdm/survey.py

```diff
@@ -475,7 +494,7 @@ def load_survey(survey_name:str, state:parameters.State, dmvals:np.ndarray,
     # Do it
     srvy=Survey()
     srvy.name = survey_name
-    srvy.process_survey_file(os.path.join(sdir, dfile))
```

Comment:
You will soon find a conflict with one of my pull requests, where I take the last "else" to assume the Survey file is "[survey_name].dat" (although I do not test to see if such a file exists, due to laziness). This is so one does not have to change the code when using a new survey file. I don't know how conflicting merges will be handled here for the two concurrent pull requests...

Reply:
Got it.  I think things will behave ok.

But, I've set it so you have to provide `Nbeam` in this case.